internal/data/accounts.go

```diff
@@ -31,12 +31,19 @@ func (m *AccountModel) Delete(ctx context.Context, address string) error {
 	return nil
 }
 
-func (m *AccountModel) Exists(ctx context.Context, address string) (bool, error) {
-	const query = `SELECT EXISTS(SELECT stellar_address FROM accounts WHERE stellar_address = $1)`
+func (m *AccountModel) IsAccountFeeBumpEligible(ctx context.Context, address string) (bool, error) {
+	const query = `
+		SELECT 
+			EXISTS(
+				SELECT stellar_address FROM accounts WHERE stellar_address = $1
+				UNION
+				SELECT public_key FROM channel_accounts WHERE public_key = $1
+			)
+	`
```

Comment:
👋  I'm confused here... is this method saying that channel accounts can sign a FeeBumpTransaction? Fee bump transactions are used when you have a distribution account paying for the gas fees, so I don't think a Channel Account should be eligible to sign fee bump transactions.

Reply:
Addressed here: https://github.com/stellar/wallet-backend/blob/049a643b4996aa355e7b6365907d1000fa3f51d4/internal/data/accounts.go#L34-L35